stage1/calc.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8"/>
+  <head>
+    <link rel="stylesheet" type="text/css" href="../normalize.css">
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class='calculator-wrapper'>
+      <div class='container calculator-body'>
```

Comment:
all of your classnames need to be prefixed with `calculator-` so `calculator-col` `calculator-size-1` etc.